### DISCUSSION TITLE
119

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#48] [General]
# 潮流周刊第119期 - 缙云石宕

封面来源于去年大概这个时候在丽水缙云某个采石场里面拍的一小段视频，后面查了查原来叫缙云石宕，缙云在浙江内属于挺好玩的一个城市，有山有水有烧饼。

https://weekly.tw93.fun/posts/119-%E7%BC%99%E4%BA%91%E7%9F%B3%E5%AE%95

**Comment by chang8128** [comment]
挺想有个本地离线的内容管理工具，我目前用 MWEB 做笔记，也记录一些随手心得，以及学习笔记，都很不错，也支持数学公式、图片插入。
奈何缺点是每次看到喜欢的网页，面对一大堆的图片，MWEB 实在是耗时耗力，而且自己的笔记如果要发布到WEB上，还需要额外加载图床。 这几个缺点让我止步了。
不知道博主有推荐的更好的方法，或者工具没有？收费的也可以啊
